modules/compliance-operator-cli-uninstall.adoc

```diff
@@ -0,0 +1,103 @@
+// Module included in the following assemblies:
+//
+// security/compliance_operator/compliance-operator-uninstallation.adoc
+
+:_content-type: PROCEDURE
+[id="compliance-operator-uninstall-cli_{context}"]
+= Uninstalling the OpenShift Compliance Operator from {product-title} using the CLI
+
+To remove the Compliance Operator, you must first delete the objects in the namespace. After the objects are removed, you can remove the Operator and its namespace by deleting the *openshift-compliance* project.
+
+.Prerequisites
+
+* Access to an {product-title} cluster using an account with `cluster-admin` permissions.
+* The OpenShift Compliance Operator must be installed.
+
+.Procedure
+
+. Delete all objects in the namespace.
+
+.. Delete the `ScanSettingBinding` objects:
++
+[source,terminal]
+----
+$ oc delete ssb <ScanSettingBinding-name> -n openshift-compliance
+----
+
+.. Delete the `ScanSetting` objects:
++
+[source,terminal]
+----
+$ oc delete ss <ScanSetting-name> -n openshift-compliance
+----
+
+.. Obtain the `ProfileBundle` objects:
++
+[source,terminal]
+----
+$ oc get profilebundle.compliance -n openshift-compliance
+----
++
+.Example output
+[source,terminal]
+----
+NAME     CONTENTIMAGE                                                                     CONTENTFILE         STATUS
+ocp4     registry.redhat.io/compliance/openshift-compliance-content-rhel8@sha256:<hash>   ssg-ocp4-ds.xml     VALID
+rhcos4   registry.redhat.io/compliance/openshift-compliance-content-rhel8@sha256:<hash>   ssg-rhcos4-ds.xml   VALID
+----
+
+.. Delete the `ProfileBundle` objects:
```

Comment:
Before delete the profilebundle, need to delete all scansettingbinding, compliancesuite, compliancescan. Otherwise the profilebundle could not be deleted successfully.
```
Delete the `ComplianceSuite` objects:
$ oc delete suite <compliancesuite-name> -n openshift-compliance
Delete the `ComplianceScan` objects:
$ oc delete scan <compliancescan-name> -n openshift-compliance
```

Reply:
Thank you @xiaojiey ! I have updated the docs accordingly.